mbrl/env/ant_truncated_obs.py

```diff
@@ -16,18 +29,23 @@ def step(self, a):
         forward_reward = (xposafter - xposbefore) / self.dt
         ctrl_cost = 0.5 * np.square(a).sum()
         contact_cost = (
-            0.5 * 1e-3 * np.sum(np.square(np.clip(self.sim.data.cfrc_ext, -1, 1)))
+            0.5 * 1e-3 * np.sum(np.square(np.clip(self.data.cfrc_ext, -1, 1)))
         )
         survive_reward = 1.0
         reward = forward_reward - ctrl_cost - contact_cost + survive_reward
         state = self.state_vector()
-        notdone = np.isfinite(state).all() and state[2] >= 0.2 and state[2] <= 1.0
-        done = not notdone
+        notterminated = np.isfinite(state).all() and state[2] >= 0.2 and state[2] <= 1.0
+        terminated = not notterminated
         ob = self._get_obs()
+
+        if self.render_mode == "human":
+            self.render()
+
         return (
             ob,
             reward,
-            done,
+            terminated,
+            False,
```

Comment:
Do we need to add logic for environments to count number of steps for `done`, or is that handled elsewhere?

Reply:
There used to be ` TimeLimit` wrapper to take care of that. I imagine something similar is still being done here.

Reply:
This is handled by the `gymnasium.wrappers.TimeLimit` wrapper we wrap all our environments with whenever they're instantiatied.

I just dug into this a little deeper though, noticed a minor bug:
https://github.com/facebookresearch/mbrl-lib/blob/main/mbrl/util/mujoco.py#L100
In this one place, `max_episode_steps=1000` is hardcoded here, whereas everywhere else, it's `max_episode_steps=cfg.overrides.get("trial_length", 1000)` e.g. https://github.com/facebookresearch/mbrl-lib/blob/main/mbrl/util/env.py#L94

I'll file this one as an issue separately, fix it in another branch since it's an existing bug not directly related to the Gymnasium migration.

Reply:
Good catch, filing issue sounds good. Thanks!